### PR TITLE
Create sitter model

### PR DESCRIPTION
**Pull request by NadzeyaShpakouskaya**
## Summary
This PR is related to issue #19.
Create a `Sitter` model with doc comments and without tests.

### Changes
- create a model layer for the pet sitter.

### CODE REVIEW

File: Woof/Woof/Model/Sitter.swift [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    /// The surname of the sitter.
+    var surname: String
+
+    /// The phone number of the sitter.
+    var phone: String
+
+    /// The URL of the sitter's avatar image.
+    var avatarUrl: URL?
+
+    /// The biography or description of the sitter.
```
Comment (iBYarchitect) [inline Woof/Woof/Model/Sitter.swift:23]: nit:
```suggestion
    /// Additional Information about sitter.
    /// The dog walker can describe their experience with dogs, favorite places to run, and any other important details
    /// that will help them better understand how to meet the needs and desires of each owner's dog
    /// They can also add information about their special skills and experience, which can be useful during the outings.
```
I'm not sure about such a long comment.

**Review by iBYarchitect — APPROVED** [review]
LGTM

File: Woof/Woof/Model/Sitter.swift (hunk 1 above)
Comment (NadzeyaShpakouskaya) [reply Woof/Woof/Model/Sitter.swift:23]: @iBYarchitect How about this?   
/// The additional information about the sitter, like his experience with dogs, favorite places for walks, special skills, certificates, etc.

File: Woof/Woof/Model/Sitter.swift [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import Foundation
+
+/**
+ A person who can walk with pet (or several pets) and provide service of walking for pet owners.
```
Comment (julyakmita) [inline Woof/Woof/Model/Sitter.swift:4]: Nit:
maybe: 
Describes a person who....

**Review by julyakmita — APPROVED** [review]
Good job!)
LGTM

File: Woof/Woof/Model/Sitter.swift [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    /// The surname of the sitter.
+    var surname: String
+
+    /// The phone number of the sitter.
+    var phone: String
+
+    /// The URL of the sitter's avatar image.
+    var avatarUrl: URL?
+
+    /// The additional information about the sitter, like his experience with dogs, favorite places for walks,
```
Comment (dersim-davaod) [inline Woof/Woof/Model/Sitter.swift:23]: nit: girls can be sitter too :)
Please use gender-neutral words where it is possible.

```suggestion
    /// The additional information about the sitter, like their experience with dogs, favorite places for walks,
```

**Review by dersim-davaod — APPROVED** [review]
LGTM

**Pull request merged**